src/lib/updateGrantsGrantees.js

```diff
@@ -90,9 +90,6 @@ export async function processFiles() {
   }
 }
 
-// reading archives
-const zip = new AdmZip('./hses.zip');
```

Comment:
I _think_ having this outside the function was leading to stale data being loaded. Moving it is part of my experiment to verify that or not, along with adding those files to `.cfignore`